packages/cloud-cognitive/src/components/AddSelect/AddSelectFilter.js

```diff
@@ -112,15 +112,18 @@ export let AddSelectFilter = ({
               <p>{filtersLabel}</p>
               <div className={`${blockClass}-opts`}>
                 {filterOpts.map((filterOpts) => (
-                  <Dropdown
-                    id={filterOpts.id}
-                    key={filterOpts.id}
-                    titleText={filterOpts.label}
-                    items={filterOpts.opts}
-                    onChange={(value) => onchangeHandler(value, filterOpts.id)}
-                    selectedItem={getSelectedItem(filterOpts.id)}
-                    label={placeholder}
-                  />
+                  <Layer key={filterOpts.id}>
```

Comment:
Does the layer need to be inside the map function?

Reply:
my understanding about how layer works is that it needs to surround each individual dropdown https://react.carbondesignsystem.com/?path=/story/components-dropdown--with-layer

Reply:
actually sorry i think i misunderstood the implementation. making some changes and then pushing an update

Reply:
I don’t think it needs to wrap each individual dropdown. I’d consider it more like a `div` that wraps a set of dropdowns on the same “layer.”